README.md

```diff
@@ -1,17 +1,19 @@
 # zati
 
+![npm version](https://img.shields.io/npm/v/@f1v/zati?color=green) [![semantic-release](https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg)](https://github.com/semantic-release/semantic-release)
+
```

Comment:
Here is a preview of the readme
![Screen Shot 2020-07-24 at 9 20 37 AM](https://user-images.githubusercontent.com/39889198/88395420-079d7b00-cd8f-11ea-9928-0b8604d4fc1a.png)